lib/linguist/languages.yml

```diff
@@ -3387,9 +3389,8 @@ Perl 6:
   - ".pl6"
   - ".pm"
   - ".pm6"
+  - ".pod6"
```

Comment:
This was discussed before, and should be omitted. See #3366.

Reply:
Checking it out... And I would kindly ask you to reconsider. What brought me (back) here was github/markup#1173, where I spent quite a long time until I discovered .pod6 files were not recognized as Perl6, which had me do a workaround... In fact, I fear .pod6 files will not be recognized as such, even by markup, if that extension is not included here. pod6 is used extensively in https://github.com/perl6/doc, if nowhere else, and as indicated in #3366, it would help adoption of the language. I can wait for that one to be accepted, though... Or just eliminate that change and defer to that one.

Reply:
Personally, I'm okay with adding `.pod6` as an extension. But since there's been no official consensus on the matter, it might be easier to omit this particular addition and defer to #3366 instead. =)